source/cloud/gcp/gke.md

```diff
@@ -52,12 +66,9 @@ Verify that the NVIDIA drivers are successfully installed.
 
 ```console
 $ kubectl get po -A --watch | grep nvidia
-kube-system   nvidia-driver-installer-6zwcn                                 1/1     Running   0         8m47s
-kube-system   nvidia-driver-installer-8zmmn                                 1/1     Running   0         8m47s
-kube-system   nvidia-driver-installer-mjkb8                                 1/1     Running   0         8m47s
-kube-system   nvidia-gpu-device-plugin-5ffkm                                1/1     Running   0         13m
-kube-system   nvidia-gpu-device-plugin-d599s                                1/1     Running   0         13m
-kube-system   nvidia-gpu-device-plugin-jrgjh                                1/1     Running   0         13m
+kube-system          nvidia-gpu-device-plugin-medium-cos-h5kkz                       2/2     Running   0          3m42s
```

Comment:
This looks very different to what it looked in the past, so I thought of updating it. Happy to remove if we want to